daft/expressions/expressions.py

```diff
@@ -3010,15 +3011,17 @@ def value_counts(self) -> Expression:
         """
         return Expression._from_pyexpr(native.list_value_counts(self._expr))
 
-    def count(self, mode: CountMode = CountMode.Valid) -> Expression:
+    def count(self, mode: str | CountMode = CountMode.Valid) -> Expression:
```

Comment:
Use `Literal["all", "valid", "null"]` instead to be more specific about what values can be passed in

Reply:
Sounds good, done